chapters/custom-license-namespaces.md

```diff
@@ -0,0 +1,194 @@
+2. the namespace maintainer creates
+   an SPDX document defining license texts
+   and corresponding identifiers
+   which include the namespave,
+   as described below;
+3. the namespace and the SPDX document
+   are registered with the SPDX project; and
+4. identifiers in the format described below
+   can then be used by anyone,
+   as long as they are defined in the SPDX document
+   available at the registered URL.
+
+
+## Z.3 License name format
+
+The following sections
+define the two formats
+for defining custom license identifiers.
+
+## Z.3.1 DNS name format
+
+A valid namespace using the DNS name format
+is a standard order DNS name,
+such as `example.com`.
+This DNS name
+must only use characters permitted by the license expression syntax
+(i.e., letters, digits, "-" and ".").
+It must not contain a period followed by a hyphen,
+as that denotes the end of the namespace.
+
+Each custom license identifier under this namespace
+consists of the following, in order:
+1. the prefix `LicenseRef-`
+2. a period
+3. the namespace
+4. a period and a hyphen
+5. any other characters permitted by the license expression syntax (i.e., letters, digits, "-" and ".")
+
+For example, for the `example.com` namespace,
+the following are identifiers:
+* `LicenseRef-.example.com.-ABC-1.0`
+* `LicenseRef-.example.com.-EULA-v3.1`
+* `LicenseRef-.example.com.-anything`
+
+## Z.3.2 Organization name format
+
+A valid namespace using the organization name format
+is the organization's name,
+such as `ExampleCorp`.
+This organization name
+must only use characters permitted by the license expression syntax
+(i.e., letters, digits, "-" and ".").
+It must not contain a double hyphen,
+as that denotes the end of the namespace.
+
+Each custom license identifier under this namespace
+consists of the following, in order:
+1. the prefix `LicenseRef-`
+2. a hyphen
+3. the namespace
+4. two hyphens
+5. any other characters permitted by the license expression syntax (i.e., letters, digits, "-" and ".")
+
+For example, for the `ExampleCorp` namespace, the following could be valid identifiers:
+* `LicenseRef--ExampleCorp--ABC-1.0`
+* `LicenseRef--ExampleCorp--EULA-v3.1`
+* `LicenseRef--ExampleCorp--anything`
+
+
+## Z.5 License definitions
+
+All the licenses in a namespace
+have to be defined in an SPDX document.
+This annex does not provide any additional mechanism
+for this task,
+as the standard
+[Other licensing information detected](./other-licensing-information-detected.md) sections
+of an SPDX document
+can be used without any special consideration.
+
+The SPDX document should not define
+any licenses with identifiers
+other than those in its own namespace.
+
+
+## Z.6 Namespace registration
+
+In order for a namespace to be usable
+in custom license identifiers,
+it must be registered with the SPDX project.
+The registration request must provide
+the namespace and
+the SPDX document defining the licenses.
```

Comment:
See my main comments about registering the document itself, vs. a URL to the document.